src/Aspire.Dashboard/DashboardWebApplication.cs

```diff
@@ -90,6 +90,23 @@ public DashboardWebApplication()
             logger.LogInformation("OTLP server running at: {OtlpEndpointUri}", reportedOtlpUri.AbsoluteUri.TrimEnd('/'));
         }
 
+        // Redirect browser directly to /StructuredLogs address if the dashboard is running without a resource service.
+        // This is done to avoid immediately navigating in the Blazor app.
+        _app.Use(async (context, next) =>
+        {
+            if (context.Request.Path == TargetLocationInterceptor.ResourcesPath)
```

Comment:
I know `ResourcesPath` is just `/`, but adding and using a `StringComparers`/`StringComparisons` entry for URI paths would avoid the sense that this might be buggy.